app/javascript/app/utils/graphs.js

```diff
@@ -102,6 +102,19 @@ export const getColorPalette = (colorRange, quantity) =>
 
 export const darkenColor = color => chroma(color).darken();
 
+export function setYAxisDomain() {
+  return [setBuffer, dataMax => dataMax];
+}
+
+function setBuffer(min) {
```

Comment:
It would be nice to have an example of when these buffers apply. Could we check it when you come to the office?

Reply:
@alvaro I've addressed your comments in the PR (moved the floating point values from the axis to the tooltip and used the string instead of the function on the domain).   
About the `buffer` it adds some space under the smaller value of the line chart to avoid confusion with the zero value (it adds a 30% percent of the minimal value on the bottom of it).

You can check it here: 
http://localhost:3000/ghg-emissions?breakBy=location&filter=GBR%2CZAF&source=10&version=1

https://climate-watch.vizzuality.com/ghg-emissions?breakBy=location&filter=ZAF%2CGBR&source=63&version=1

Reply:
![image](https://user-images.githubusercontent.com/9701591/42156953-b4583bfe-7dec-11e8-9f9e-e31876f6425b.png)
I can't see the buffer working. Lets check it when you are back

Reply:
It is working (otherwise the lower value of the chart would stick to the `x-axis`), what happens is that because of the great difference between higher and lower chart values the buffer seems too small. Maybe would be a good fix to calculate the buffer taking into account higher and lower values.
Let's discuss the approach having in mind that [this task](https://basecamp.com/1756858/projects/13795275/todos/352513391) does not have a clear definition of whats the intended behaviour.

Reply:
If we lack clarity let's not dwell too much on it. Fix the key issues and then when replying to them ask for further clarity if they have it or say that we are happy to discuss in more detail if that helps.